VueJS/README.md

```diff
@@ -1,17 +1,9 @@
-# Exercise \#1
+# ToDo List with VueJS
 
-Implement a simple todo app with VueJS.
+To execute:
 
-Each item in the todo app should:
+* Install npm
+* Install vue cli: "npm install -g @vue/cli"
```

Comment:
```suggestion
```
In fact, you don't have to install `@vue/cli` globally. Both package managers, `npm` and `yarn` install executables into a folder `node_modules/.bin/` and if you run the command through `[npm|yarn] run <executable>` then it will look for the executable in that folder.